src/vs/base/parts/ipc/test/node/ipc.cp.integrationTest.ts

```diff
@@ -17,67 +17,58 @@ function createClient(): Client {
 }
 
 suite('IPC, Child Process', function () {
+	this.slow(2000);
 	this.timeout(10000);
 
-	test('createChannel', () => {
+	test('createChannel', async () => {
 		const client = createClient();
 		const channel = client.getChannel('test');
 		const service = new TestServiceClient(channel);
 
-		const result = service.pong('ping').then(r => {
-			assert.strictEqual(r.incoming, 'ping');
-			assert.strictEqual(r.outgoing, 'pong');
-		});
+		const result = await service.pong('ping');
+		assert.strictEqual(result.incoming, 'ping');
+		assert.strictEqual(result.outgoing, 'pong');
 
-		return result.finally(() => client.dispose());
+		client.dispose();
```

Comment:
Before, this was also called in case of a test failure...

Reply:
You're right! Follow up: https://github.com/microsoft/vscode/pull/165233